ansible_collections/arista/avd/roles/eos_designs/templates/connected_endpoints/ethernet-interfaces.j2

```diff
@@ -67,6 +67,11 @@ ethernet_interfaces:
     channel_group:
       id: {{ channel_group_id }}
       mode: "{{ adapter_settings.port_channel.mode }}"
+{%                     elif adapter_settings.link_tracking.enabled is arista.avd.defined(true) and switch.link_tracking_groups is arista.avd.defined %}
```

Comment:
Even though this logic works, I would prefer to have it in a separate `if` block to make it more clear. Then we don't need the comment either.

Reply:
Then I need to add if check for port-channel doesn't exist. ```if adapter_settings.port_channel is not arista.avd.defined``` and then add other logic with ```and```